### ISSUE TITLE
Enable accessibility problem

### ISSUE THREAD

**Issue by M4rteaga** [withfig/fig#1138]
### When updated the app to version 1.0.56 the wizard ask me to enable accessibility witch was already enable, so a follow the instructions and disabled it and enable it multiple times, but that doesn't worked for me. I also tried to reinstall te app, and exactly the same cycle happened. Also tried to restarting my Mac and  doesn't work either.  

https://user-images.githubusercontent.com/47699470/159137895-ff629062-2521-47c9-90b0-5f043d36cd5c.mov

### Details:
|OS|Fig|Shell|
|-|-|-|
|macOS 12.3.0 (21E230)|1.0.56|/bin/zsh|
<details><summary><code>fig diagnostic</code></summary>
<p>

# Fig Diagnostics
## Fig details:
  - Fig version: Version 1.0.56 (B414) [ABC]
  - Bundle path: /Applications/Fig.app
  - Autocomplete: true
  - Settings.json: true
  - Accessibility: false
  - Number of specs: 0
  - Symlinked dotfiles: false
  - Only insert on tab: false
  - Keybindings path: 
  - Installation Script: true
## Hardware Info:
  - Model Name: MacBook Air
  - Model Identifier: MacBookAir10,1
  - Chip: Apple M1
  - Cores: 8 (4 performance and 4 efficiency)
  - Memory: 8 GB
## OS Info:
  - macOS 12.3.0 (21E230)
## Environment:
  - User Shell: /bin/zsh
  - CLI Installed: true
  - Executable Location: /Users/{username}/.local/bin/fig
    - TERM=xterm-256color
    - FIG_INTEGRATION_VERSION=8
    - FIG_TERM=1
    - FIG_TERM_VERSION=4.3.0
## Integrations:
  - SSH: false
  - TMUX: false
  - iTerm: installed!
  - Hyper: application is not present.
  - Visual Studio Code: installed!
  - Docker: false

</p>
</details>
